### PR TITLE
update makefile and doc

### PR DESCRIPTION
**Pull request by mjudeikis**
Add secret from ci cluster

### CODE REVIEW

**Review by asalkeld — COMMENTED** [review]
maybe add 
```
source secrets/secret
```
to example.env too?

File: doc/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Debug
+
+Common problems:
+
+1. When starting RP I get an error:
+```
+InnerError={"code":"ForbiddenByPolicy"}
```
Comment (asalkeld) [inline doc/FAQ.md:7]: I am getting this :-(

Comment (mjudeikis) [reply doc/FAQ.md:7]: this means your Vault policy is wrong. Potentially when you created it the wrong env was configured.

**Comment by jim-minter** [comment]
am going to drop this for now - the debug issue is a bug which @asalkeld will hopefully submit a PR for - no doc needed.  The /secrets stuff: I'd like to avoid automatically adding the same boilerplate infrastructure as v3 - at the moment RP setup is certainly fiddly, but I'd like to explore alternative ways to structure and simplify.  We can discuss on Bluejeans.

**Pull request closed without merging**